opentelemetry-sdk/src/testing/logs/in_memory_exporter.rs

```diff
@@ -83,14 +86,24 @@ impl InMemoryLogsExporterBuilder {
     /// Creates a new instance of `InMemoryLogsExporter`.
     ///
     pub fn new() -> Self {
-        Self {}
+        Self {
+            reset_on_shutdown: true,
+        }
+    }
+
+    /// If set, the records will not be [`InMemoryLogsExporter::reset`] on shutdown.
```

Comment:
i am not sure if there is sufficient usecases to expose this new public api. I'd suggest to remove this, and only add, if there is enough demand.

Reply:
I need it for unit test. I can make it pub(crate)